tox.ini

```diff
@@ -66,7 +66,6 @@ commands =
   bash -c "! find doc/ -type f -name *.json | xargs grep -U -n $'\r'"
   # Check that all included JSON files are valid JSON
   bash -c '! find doc/ -type f -name *.json | xargs -t -n1 python -m json.tool 2>&1 > /dev/null | grep -B1 -v ^python'
-  bash tools/check-cherry-picks.sh
```

Comment:
That change seems someone unrelated to resizing and migration.
Should it really be removed?

Reply:
I just see this now. Probably I accidentally included this in one of the previous commit amends, or I need to rebase this branch. Will check.